dvc/utils/hydra.py

```diff
@@ -30,16 +31,26 @@ def compose_and_dump(
             Ignored if `config_dir` is not `None`.
         config_name: Name of the config file containing defaults,
             without the .yaml extension.
+        plugins_path: Path to auto discover Hydra plugins.
         overrides: List of `Hydra Override`_ patterns.
 
     .. _Hydra Override:
         https://hydra.cc/docs/advanced/override_grammar/basic/
     """
+    import sys
+
     from hydra import compose, initialize_config_dir, initialize_config_module
+    from hydra.core.plugins import Plugins
     from omegaconf import OmegaConf
 
     from .serialize import DUMPERS
 
+    sys.path.append(plugins_path)
+    try:
+        Plugins.instance()
+    finally:
+        sys.path.pop()
+
```

Comment:
Can we move this to a separate function, and may be call `load_hydra_plugins`? Or, maybe `load_hydra_plugins_dangerously` to make it more obvious?